azimuth/utils/string_processing.py

```diff
@@ -0,0 +1,18 @@
+# Copyright ServiceNow, Inc. 2021 – 2022
+# This source code is licensed under the Apache 2.0 license found in the LICENSE file
+# in the root directory of this source tree.
+
+
+def clean_utterance(utterance: str, lowercase: bool = True):
```

Comment:
Those are tiny details:
- I find there is a little mismatch between the super generic file name pertaining to a "string" and the quite more specific function name about "utterance". My gut feeling is that we can make the file specific to utterances. I see other existing functions that do some "string processing", but they are in defined in some specific files where they are relevant, like `to_camel_case(string: str) -> str` or `get_utterances_diff(original_utterance: str, perturbed_utterance: str) -> List[str]`. Maybe this file could be in the `ml/` directory& Like `azimuth/utils/ml/utterance.py`? What do you think?
- If we don't have a reason to use the `lowercase` flag, I would remove it. Worse case, we add it if we need it one day. Otherwise, it's dead code that will stay there forever, and one more "branch" to unit-test.
```suggestion
def clean_utterance(utterance: str):
```

Reply:
Fair points. I'll remove the lowercase parameter and rename to `utterance.py` (and we can rename it if anything else goes into it that necessitates making it more specific, etc.). I don't think it should be in the ML directory - looking at the other files there, they seem to actually be related directly to models/ML, and this is for POS-tagging and filtering. Sound ok to you?

Reply:
Sounds perfect to me!

Reply:
Perfect! Thank you!